src/react/components/shell/mainContentRouter.tsx

```diff
@@ -7,6 +7,9 @@ import ConnectionPage from "../pages/connections/connectionsPage";
 import EditorPage from "../pages/editorPage/editorPage";
 import ExportPage from "../pages/export/exportPage";
 import ProjectSettingsPage from "../pages/projectSettings/projectSettingsPage";
+import LogInForm from "../pages/signIn/signInPage";
```

Comment:
We are importing `page` not `form` so I believe that better name would be `SignInPage`